.gitignore

```diff
@@ -59,3 +59,6 @@ social_network
 # VIM swap files #
 *.swp
 *.swo
+
+# Maven auto-generated files #
+**/.factorypath
```

Comment:
@sorsaffari was meant to be removed?

Reply:
@haikalpribadi It was! I'll have it removed in my next pull request (today).